networkx/algorithms/covering.py

```diff
@@ -0,0 +1,131 @@
+#    Copyright 2016 NetworkX developers.
+#    Copyright (C) 2004-2016 by
+#    Nishant Nikhil <nishantiam@gmail.com>
+#    All rights reserved.
+#    BSD license.
+
+""" Functions related to graph covers."""
+
+import networkx as nx
+from networkx.utils import not_implemented_for, arbitrary_element
+
+__all__ = ['min_edge_cover', 'is_edge_cover', 'is_min_edge_cover']
+
+
+@not_implemented_for('directed')
+@not_implemented_for('multigraph')
+def min_edge_cover(G, matching_algorithm=None):
+    r"""Returns a set of edges which constitutes
```

Comment:
All the same comments about the docstring apply here as for the one in `bipartite`.